### PR TITLE
LPD-15432 Stop using org.mvel:mvel2

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPD-15432](https://liferay.atlassian.net/browse/LPD-15432)

### CODE REVIEW

**Review by p-albuquerque — APPROVED** [review]

File: modules/third-party/org-mvel2/build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ sourceCompatibility = "1.6"
 targetCompatibility = "1.6"
 
 dependencies {
-	api group: "org.mvel", name: "mvel2", version: "2.1.0.drools16"
+	api group: "com.liferay", name: "org.mvel2", version: "2.1.0.drools16.LIFERAY-PATCHED-2"
```
[the remark this answers is not in the document]
Comment (MarinhoFeliphe) [reply modules/third-party/org-mvel2/build.gradle:11]: No need to change, I've not noticed that according to [this comment](https://liferay.atlassian.net/browse/LPD-15432?focusedCommentId=2620463) your changes are exactly what you should do

Thank you!

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by MarinhoFeliphe** [comment]
ci:help

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
Resent here : https://github.com/brianchandotcom/liferay-portal/pull/148980

**Pull request closed without merging**